domain.yml

```diff
@@ -511,6 +509,11 @@ templates:
       buttons:
         - title: "Continue"
           payload: "/get_started_step3"
+  utter_direct_to_step2:
+    - text: "If you have no questions, you can click the button below to go to the next step once you're ready. Or when you're finished asking questions, you can say 'take me to the next step' at any point"
```

Comment:
let's leave out the "if you have no questions" and "take me to the next step thing" -- i think it only really makes sense to have that in step2